i18n/de.toml

```diff
@@ -49,13 +53,13 @@ delete = "Fahrzeug löschen"
 generic = "Weitere Integrationen"
 offline = "Generisches Fahrzeug"
 online = "Fahrzeuge mit Schnittstelle"
-save = "Speichern"
+save = Speichern
```

Comment:
@premultiply @naltatis warum sind einige Einträge in Anführungszeichen und viele nicht? Müssen wir da noch was am Tooling verbessern?

Reply:
Das fixt mein regelmäßiger `make toml` Lauf.